pom.xml

```diff
@@ -59,6 +76,18 @@
       <artifactId>marc4j</artifactId>
       <version>2.9.1</version>
     </dependency>
+    <dependency>
+      <groupId>org.assertj</groupId>
+      <artifactId>assertj-core</artifactId>
+      <version>3.19.0</version>
+      <scope>test</scope>
+    </dependency>
```

Comment:
This project already uses Hamcrest for matchers, why has an alternative been introduced?

Reply:
The https://github.com/folio-org/raml-module-builder/blob/master/doc/upgrading.md#version-330 says: "Add org.assertj test dependency to pom.xml if Assertj is used. RMB no longer ships with Assertj."

This pull request doesn't add any additional Assertj tests. The pom.xml entry is needed for the existing Assertj tests.

I've created https://issues.folio.org/browse/MODINVSTOR-734 "Replace Assertj matchers by Hamcrest matchers" = https://github.com/folio-org/mod-inventory-storage/pull/630 to fix this technical debt.